kura/org.eclipse.kura.rest.service.listing.provider/OSGI-INF/ServiceListingRestService.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Eurotech and/or its affiliates and others
+  
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+ 
+    SPDX-License-Identifier: EPL-2.0
+    
+    Contributors:
+     Eurotech
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.kura.internal.rest.service.listing.provider.RestServiceListingProviderstingProvider">
+    <implementation class="org.eclipse.kura.internal.rest.service.listing.provider.RestServiceListingProvider"/>
+
+    <property name="kura.service.pid" type="String" value="org.eclipse.kura.internal.rest.service.listing.provider.RestServiceListingProvider"/>
+
+    <reference interface="org.eclipse.kura.cloudconnection.request.RequestHandlerRegistry"
+               bind="bindRequestHandlerRegistry"
+               unbind="unbindRequestHandlerRegistry"
+               cardinality="0..n"
+               name="RequestHandlerRegistry"
+               policy="dynamic"/>
+
+    <reference interface="org.osgi.service.useradmin.UserAdmin"
```

Comment:
We can remove the specific `kura.permission.rest.serviceListing` role and allow access to this rest API to any authenticated user. If this is done this reference is not required.